README.md

```diff
@@ -1,38 +1,7 @@
-## Edit the character files
+### WHAT THIS REPO IS
+This repo uses the ElizaOS Agentic AI framework to create an AI agent that interacts with your input (via terminal or twitter!).
+You give the agent a gift code and your wallet address and it will mint a gift NFT for you on the Avalanche Fuji Network.
 
-Open `src/character.ts` to modify the default character. Uncomment and edit.
+The agent uses custom actions that interact with [Chainlink Functions](https://docs.chain.link/chainlink-functions).
```

Comment:
Perhaps hyperlink to the actions docs in ElizaOS docs so that people know what we mean by "Actions"

Reply:
Good point. Updated